src/saving/QuickLoad.tscn

```diff
@@ -0,0 +1,20 @@
+[gd_scene load_steps=2 format=2]
+
+[ext_resource path="res://src/gui_common/QuickLoadHandler.cs" type="Script" id=1]
```

Comment:
The scene file should be named the same as the scn.
So please move and rename this scene file to be in `src/gui_common/QuickLoadHandler.tscn`